.gitignore

```diff
@@ -2,4 +2,5 @@ Cargo.lock
 target/
 bin/*.after
 bin/*.before
-bin/*.o
\ No newline at end of file
+bin/*.o
+*.swp
```

Comment:
I think this line belongs in you user-wide git ignore

Reply:
Removed *.swp